src/Stack2nix/External/Stack.hs

```diff
@@ -142,10 +146,11 @@ planAndGenerate boptsCli baseDir outDir remoteUri revPkgs argRev threads doAfter
   plan <- withLoadPackage $ \loadPackage ->
     constructPlan mbp baseConfigOpts locals extraToBuild localDumpPkgs loadPackage sourceMap installedMap (boptsCLIInitialBuildSteps boptsCli)
   let pkgs = prioritize $ planToPackages plan ++ revPkgs
-  -- liftIO $ hPutStrLn stderr $ "plan:\n" ++ ppShow pkgs
+  liftIO $ hPutStrLn stderr $ "plan:\n" ++ show pkgs
 
-  void $ liftIO $ mapM_ (\pkg -> cabal2nix ("cabal://" ++ pkg) Nothing Nothing (Just outDir)) $ words "hscolour stringbuilder"
-  void $ liftIO $ mapPool threads (genNixFile baseDir outDir remoteUri argRev) pkgs
+  hackageDB <- liftIO $ loadHackageDB Nothing hSnapshot
+  void $ liftIO $ mapM_ (\pkg -> cabal2nix ("cabal://" ++ pkg) Nothing Nothing (Just outDir) hackageDB) $ words "hscolour stringbuilder"
```

Comment:
I want to kill this hack. Some packages will build if more hardcoded packages are added here, but ideally there shouldn't be any.